lib/mvi/mock_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module MVI
+  class MockService
+    def self.mocked_responses
+      @responses ||= YAML.load_file("#{Rails.root}/config/mvi_schema/mock_mvi_responses.yml")
+    end
```

Comment:
remind me again why Rails.root is necessary? paths should be relative to the automatically I think.